editor/src/clj/editor/gui.clj

```diff
@@ -801,10 +803,12 @@
                     :child-index child-index
                     :layer-index layer-index
                     :topmost? true
+                    :visible-self? (and visible enabled)
+                    :visible-children? enabled
```

Comment:
These two booleans are now processed by the `scene-view` as it flattens the renderables. They will hide the object in the same way as hiding it from the View menu does. This results in only rendering the outline or selection hit test pass for the renderable if it is currently selected, otherwise we render nothing. We need the selection hit test shapes to be able to drag the selected objects around, even though they are only rendered as outlines.